src/utils/jira.py

```diff
@@ -82,30 +82,19 @@ def preprocess(
 def get_fields_ids(
     jira_client: jira.client.JIRA, issues: list[jira.resources.Issue]
 ) -> dict[str, str]:
-    ids = {}
     all_the_fields = jira_client.fields()
     # print(dir(issues[0].fields))
     # debug_fields = '\n'.join(sorted([i['name'] for i in all_the_fields]))
     # print(f"Fields:\n{debug_fields}")
+    return dict([(field["name"], field["id"]) for field in all_the_fields])
 
-    ids["Rank"] = [f["id"] for f in all_the_fields if f["name"] == "Rank"][0]
-    ids["Target Start Date"] = [
-        f["id"] for f in all_the_fields if f["name"] == "Target start"
-    ][0]
-    ids["Target End Date"] = [
-        f["id"] for f in all_the_fields if f["name"] == "Target end"
-    ][0]
-    ids["Due Date"] = [f["id"] for f in all_the_fields if f["name"] == "Due Date"][0]
-
-    return ids
 
-
-def get_parent(jira_client: jira.client.JIRA, issue: jira.resources.Issue):
-    query = f'issue in parentIssuesOf("{issue.key}")'
```

Comment:
The `field_id` things was brittle for me.

I can reproduce the issue you're trying to solve with PLNSRVCE-1006, the following query seems to fix it:
`issue in parentIssuesOf("{issue.key}") AND issue in linksHierarchyIssue("{issue.key}") `

Reply:
Happy to solve it with a modified query, but - how was field_ids brittle?

Reply:
See https://github.com/ralphbean/prioritize/pull/20 which uses your proposed query.

Reply:
I think it was breaking with the old getChildren query when there was no parent. So it might not have been the field that brittle rather than our usage of it.

I still find the updated query a more elegant solution though, as it does not require additional data, and should continue to work in the case of an unlikely change to fields name (or their existance).